v2/datastream-to-spanner/terraform/samples/monitoring-alerting/main/.gitignore

```diff
@@ -0,0 +1,48 @@
+# Terraform state files
```

Comment:
Do we need this? I think there is a global gitignore that we can use instead?

Reply:
Agree, we do not need this. I have removed this file in the new commit.